app/models/miq_ae_namespace.rb

```diff
@@ -72,9 +73,8 @@ def self.find_or_create_by_fqname(fqname, include_classes = true)
   # TODO: broken since 2017
   def self.find_tree(find_options = {})
     namespaces = where(find_options)
-    ns_lookup = namespaces.inject({}) do |h, ns|
-      h[ns.id] = ns
-      h
+    ns_lookup = namespaces.index_by do |ns|
+      ns.id
     end
```

Comment:
expecting to see:

```ruby
ns_lookup = namespaces.index_by(&:id)
```

Reply:
yeah, the conversion of code from block to `&:` is a separate cop that I didn't run - I'm ok leaving this as is for now.